### PR TITLE
Add improvements.

### PR DESCRIPTION
**Pull request by dewniMW**

### CODE REVIEW

File: en/docs/guides/password-mgt/challenge-question.md [hunk 1]
```diff
@@ -1,17 +1,183 @@
...
+    
+2. Navigate to **Main** > **Identity** > **Users and Roles** > **Add**.
+    
+3. Click **Add New User** and create a new user with the username "Alex".
+    
+4. Click **Finish**.
+
+## Enable password reset via Challenge Questions
+
+Follow the steps below to configure WSO2 Identity Server to enable password reset by answering to a challenge question.
```
Comment (gomathyk) [inline en/docs/guides/password-mgt/challenge-question.md:19]:
```suggestion
Follow the steps below to configure WSO2 Identity Server to enable password reset by answering a challenge question.  
```

File: en/docs/guides/password-mgt/challenge-question.md [hunk 2]
```diff
@@ -1,17 +1,183 @@
...
+	
+	![security-question-based-password-recovery-option](../../../assets/img/guides/security-question-based-password-recovery-option.png)
+
+5.	Click **Update**. 
+
+----
+
+## Configure the challenge questions
+
+1. Access WSO2 Identity Server My Account (`https://<HOST>:<PORT>/myaccount`) application.
```
Comment (gomathyk) [inline en/docs/guides/password-mgt/challenge-question.md:45]:
```suggestion
1. Access the WSO2 Identity Server My Account (`https://<HOST>:<PORT>/myaccount`) application.		
```

File: en/docs/guides/password-mgt/challenge-question.md [hunk 3]
```diff
@@ -1,17 +1,183 @@
...
+
+## Configure the challenge questions
+
+1. Access WSO2 Identity Server My Account (`https://<HOST>:<PORT>/myaccount`) application.		
+
+2.	Log in with the credentials of the user account that you created.
+
+	![myaccount-login](../../../assets/img/fragments/myaccount-login.png)
+ 
+3. Select `Security` tab.
```
Comment (gomathyk) [inline en/docs/guides/password-mgt/challenge-question.md:51]:
```suggestion
3. Select the **Security** tab.
```

File: en/docs/guides/password-mgt/challenge-question.md [hunk 4]
```diff
@@ -1,17 +1,183 @@
...
+5. Click **Save**.
+
+6. Sign out.  
 
 ---
 
-## Manage challenge questions using REST APIs
+## Recover password using the My Account application
+
+1. Access WSO2 Identity Server My Account (`https://<HOST>:<PORT>/myaccount`) application.
```
Comment (gomathyk) [inline en/docs/guides/password-mgt/challenge-question.md:76]:
```suggestion
1. Access the WSO2 Identity Server My Account (`https://<HOST>:<PORT>/myaccount`) application.
```

File: en/docs/guides/password-mgt/challenge-question.md [hunk 5]
```diff
@@ -1,17 +1,183 @@
...
+
+2.	Click **Password**.
+
+    ![forgotten-password-option](../../../assets/img/guides/forgotten-password-option.png)
+
+3.	Enter the user name as `Alex` and if multiple recovery options are displayed, select the **Recover with Security Questions** option.
```
Comment (gomathyk) [inline en/docs/guides/password-mgt/challenge-question.md:82]: IMO, we should not be giving users the definite values to be entered. It becomes more like a quick start that way. 

Shall we give more generic instructions and then maybe have a try it out section where we give this as an example? WDYT?

File: en/docs/guides/password-mgt/challenge-question.md [hunk 6]
```diff
@@ -1,17 +1,183 @@
...
+6.	Enter the second challenge question answer as `Pizza` and click **Submit**.
+
+	![security-question-food](../../../assets/img/guides/security-question-food.png)  
 
-There are a number of operations related to challenge questions that you can perform using REST APIs. To manage the challenge questions and answers of a user using REST APIs, see [Manage Challenge Questions](TODO:insert-link).
+7.	Enter the new password and click **Proceed**.
+
+    ![password-reset-form](../../../assets/img/guides/password-reset-form.png) 
+
+8.	Enter the user name and new password and click **Sign In**. The My Account home screen appears.
```
Comment (gomathyk) [inline en/docs/guides/password-mgt/challenge-question.md:100]:
```suggestion
8.	Enter the username and new password and click **Sign In**. The My Account home screen appears. 
```

File: en/docs/guides/password-mgt/recover-password.md [hunk 7]
```diff
@@ -1,87 +1,110 @@
...
+	priority= "95"
+	enable = true
+	[event.default_listener.governance_identity_store]
+	priority= "97"
+	enable = true
+	```
 
-### Send recovery notification
+	!!! info 
+        This is already configured this way by default. You can skip this step if you have not changed this configuration previously.
```
Comment (gomathyk) [inline en/docs/guides/password-mgt/recover-password.md:41]: Please check if this alignment appears properly.

File: en/docs/guides/password-mgt/recover-password.md [hunk 8]
```diff
@@ -1,87 +1,110 @@
...
+5.	Under the **Account Management** section, click **Account Recovery**.
 
-```curl tab="Sample Response" 
-"HTTP/1.1 202 Accepted"
-```
+    ![account-recovery-option](../../../assets/img/fragments/account-recovery-option.png) 
 
-### Get challenge question of user
-This API is used to initiate password recovery using user challenge questions, one at a time. Response will be a random challenge question with a confirmation key.
+6.	Select **Notification based password recovery** check box.
```
Comment (gomathyk) [inline en/docs/guides/password-mgt/recover-password.md:53]:
```suggestion
6.	Select **Notification based password recovery**.
```

File: en/docs/guides/password-mgt/recover-password.md [hunk 9]
```diff
@@ -1,87 +1,110 @@
...
-curl -X GET -H "Authorization: Basic YWRtaW46YWRtaW4=" -H "Content-Type: application/json"  "https://localhost:9443/api/identity/recovery/v0.9/security-question?username=kim"
-```
+---
+       
+## Recover password using the My Account application 
 
-```curl tab="Sample Response"
-{"key":"7ced9ef0-7f3f-4f65-a115-ddbcce3a6b49","question":{"question":"Place of birth ?","question-set-id":"http://wso2.org/claims/challengeQuestion1"}
-```
+1. Access WSO2 Identity Server My Account (`https://<HOST>:<PORT>/myaccount`) application.
```
Comment (gomathyk) [inline en/docs/guides/password-mgt/recover-password.md:63]:
```suggestion
1. Access the WSO2 Identity Server My Account (`https://<HOST>:<PORT>/myaccount`) application.
```

**Review by gomathyk — CHANGES_REQUESTED** [review]

File: en/docs/guides/password-mgt/recover-password.md (hunk 7 above)
Comment (dewniMW) [reply en/docs/guides/password-mgt/recover-password.md:41]: yes it appears properly.

File: en/docs/guides/password-mgt/challenge-question.md (hunk 5 above)
Comment (dewniMW) [reply en/docs/guides/password-mgt/challenge-question.md]: removed 'Alex'

**Review by gomathyk — APPROVED** [review]

**Pull request merged**